homeassistant/components/sensor/recorder.py

```diff
@@ -115,6 +119,19 @@
         VOLUME_CUBIC_METERS: lambda x: x,
         VOLUME_CUBIC_FEET: volume_util.cubic_feet_to_cubic_meter,
     },
+    # Convert volume to L
```

Comment:
Should volume_util be adjusted to default to cubic meters?

Reply:
Yeah, I think we should convert to the SI-unit which is cubic meter, not liter.

Reply:
I have created #78861 to adjust this in the volume utility